### ISSUE TITLE
[Accessibility] Play button for cloud native program video on home page is an image without an alt tag

### ISSUE THREAD

**Issue by OMGATE23** [layer5io/layer5#4266]
### Description
The video play button is an image without an alt. This will cause accessibility issues and will not allow tools like screen readers to understand what the image tag is and if it is a button to play a video.

I would like to work on this issue

### Expected Behavior
The image should have an alt attribute as well as aria-role set as button to improve the accessibility for the play button.

### Screenshots
Following is the play button we are talking about:
![image](https://github.com/layer5io/layer5/assets/90276220/4ba509ae-a057-4c5c-a6a4-564f250529fd)

Following is what lighthouse has to say about the issue:
![play button](https://github.com/layer5io/layer5/assets/90276220/54acbfaa-e35b-456c-9b98-72ffeea954d0)

### Environment:
- Host OS: Windows
- Browser: Brave

---
<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.png" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

**Comment by OMGATE23** [comment]
I would like to work on this issue

**Comment by Savio629** [comment]
// @ayushthe1

**Comment by surajvast1** [comment]
i would like to work on this

**Comment by Shivam-AfA** [comment]
Oh no, I think the assignment request for @OMGATE23 was bypassed by mistake. There are many other issues to work on @OMGATE23. Thanks for opening this issue.

P.S. : @ayushthe1 is also into the lighthouse stuff. You can collaborate with him o the improvment of the lighthouse scores.
